Neuro/Impostor/ImpostorHandler.cs

```diff
@@ -0,0 +1,243 @@
+﻿using UnityEngine;
+using Reactor.Utilities.Attributes;
+using System;
+using System.Collections.Generic;
+using Neuro.Vision.DataStructures;
+using BepInEx.Unity.IL2CPP.Utils;
+using System.Collections;
+using Il2CppInterop.Runtime.Attributes;
+using System.Linq;
+
+namespace Neuro.Impostor;
+
+[RegisterInIl2Cpp]
+public class ImpostorHandler : MonoBehaviour
+{
+    public ImpostorHandler(IntPtr ptr) : base(ptr) { }
+
+    public bool isImpostor { get; set; } = false;
```

Comment:
Don't save this in a property, always check `PlayerControl.LocalPlayer.Data.Role.IsImpostor` instead.